ArduCopter/RC_Channel.cpp

```diff
@@ -111,6 +111,7 @@ void RC_Channel_Copter::init_aux_function(const aux_func_t ch_option, const aux_
     case AUX_FUNC::SUPERSIMPLE_MODE:
     case AUX_FUNC::SURFACE_TRACKING:
     case AUX_FUNC::WINCH_ENABLE:
+    case AUX_FUNC::DISARM:
```

Comment:
Everything here is in alphabetical order.  We ought to not mess that up.

Reply:
fixed it!